cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -783,3 +793,245 @@ def test_search_disabled(self, store_type):
     @ddt.data(*WORKS_WITH_STORES)
     def test_exception(self, store_type):
         self._perform_test_using_store(store_type, self._test_exception)
+
+
+class GroupConfigurationSearchMongo(CourseTestCase, MixedWithOptionsTestCase):
```

Comment:
If we're inheriting from the `MixedWithOptionsTestCase` then we have both stores defined within the test case anyway. Wondering why you're not following the same pattern (using ddt) as the other TestCases that inheirit from `MixedWithOptionsTestCase`